client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -362,6 +363,19 @@ export const PatternLibrary = ( {
 							patterns={ patterns }
 							patternTypeFilter={ patternTypeFilter }
 						/>
+
+						{ searchTerm && ! patterns.length && category && (
+							<div>
```

Comment:
Could we center the button and maybe give the wrapper some vertical padding..? Something like this:

```css
text-align: center;
padding: 48px 0 24px;
```